tests/models/test_tpu.py

```diff
@@ -486,3 +486,31 @@ def teardown(self, stage):
 
     model = DebugModel()
     tpipes.run_model_test(trainer_options, model, on_gpu=False, with_hpc=False)
+
+
+@RunIf(tpu=True)
+@pl_multi_process_test
+def test_tpu_spawn_debug_plugin(tmpdir):
+    """Test if `tpu_spawn_debug` plugin works."""
+
+    class DebugModel(BoringModel):
+
+        def on_train_start(self):
+            assert os.environ.get("PT_XLA_DEBUG") == str(1), "PT_XLA_DEBUG was not set in environment variables"
+
+        def teardown(self, stage):
+            assert "PT_XLA_DEBUG" not in os.environ
```

Comment:
did you put an assert False into these hooks to check that they run? 
xD
just saying, because there are no other asserts in the test here to be sure :)

Reply:
@awaelchli A lot of assertions happening in `tpipes.run_model_test` ! :D

Reply:
aha oki doki